specification/cognitiveservices/data-plane/TextAnalytics/v2.0/TextAnalytics.json

```diff
@@ -105,14 +105,6 @@
             "schema": {
               "$ref": "#/definitions/BatchInput"
             }
-          },
```

Comment:
This could be a breaking change, please acknowledge

Reply:
The service is still accepting this parameter but deprecation has been known for a while now and this parameter is no longer being "processed". If users are already using this parameter in their code then yes, it's a breaking change.